consent_manager/js_api/installation.rst

```diff
@@ -0,0 +1,61 @@
+Installation
+------------
+Consent Manager is fully integrated with Tag Manager. If you have already installed asynchronous snippet and you are using API only from Tag Manager tags, you are able use JavaScript API without aby pitfalls.
+
+Only one thing should be considered before using API is where you call commands. If your goal is to perform API method outside Tag Manager tags like in below example:
+
+.. code-block:: html
+
+    <!DOCTYPE html>
+    <html>
+        <head>
+        <meta charset="UTF-8">
+        <title></title>
+    </head>
+
+    <!-- Start Piwik PRO Tag Manager code -->
+    <script>
+        // Tag Manager async code snippet
+    </script>
+    <!-- End Piwik PRO Tag Manager code -->
+
+    <script>
+        // API call outside Tag Manager injected manually
+        ppms.cm.api(command, ...args);
+    </script>
+
+    <body>
+        Rest content of document
+    </body>
+
+    </html>
+
+When you need execute API in such manner, you should take care about Tag Manager snippet version.
+Because `ppms.cm.api` global object is initialized in snippet and/or in Tag Manager container, if you are using old version of Tag Manager snippet (PPMS version > 6.2), `ppms.cm.api` might be undefined until container will be loaded.
```

Comment:
Shouldn't it be `PPMS version < 6.2`? I'm guessing based on context.

Reply:
correct, fixed